src/rulesets/operations.ts

```diff
@@ -155,6 +155,25 @@ export const rules = {
       },
     );
   },
+  orgOrGroupTenant: ({ operations }: SnykApiCheckDsl) => {
```

Comment:
So moving this from specifications -> operations makes it magically work? Nice!

Reply:
In order to make this rule operate on a delta rather than make it an invariant that applies to all versions, released or not, I need to make it "hang off of" some element that can change in Optic's model of the OpenAPI spec. You're not going to add a new path that does anything without operations in it (hmm, is that a rule? might need to check on this...) in it, so operations seems a natural place to catch it.The other stuff in specifications seem to be invariants.

Moving it was the easy part -- most of my time on this was spent trying to figure out how to write my Jest tests and get them passing 😅.